src/Utils.cs

```diff
@@ -93,5 +94,10 @@ public static void LowercasePropertyNames(this JObject obj)
                 property.Replace(new JProperty(property.Name.ToLowerInvariant(), property.Value));
             }
         }
+
+        public static void LogDebugWithThreadId(this ILogger logger, string message, params object[] args)
```

Comment:
@Charles-Gagnon I included the thread Id but not sure if it is actually useful for input output bindings - it was mainly for row locks in trigger code right?

Reply:
Yeah it was, but in general it's not a bad thing to add - especially as we are looking into supporting scalable apps.